docs/contrib/keycloak.md

```diff
@@ -0,0 +1,148 @@
+# Keycloak Identity Provider for REMS
+
+Using docker, docker-compose, and Keycloak, you can roll your own local IdP for use with REMS.
+
+## Quick-ish Start
+
+1. **Setting up the Docker files**
+    1. Create a `Dockerfile` for your keycloak. To start, it can be as simple as the following:
+    ```
+    FROM quay.io/keycloak/keycloak:11.0.2
+    ```
+    2. Create a `docker-compose.yaml` file for REMS and your Keycloak IdP. Mount your custom REMS config to `/rems/config/config.edn`. Example:
+    ```
+    version: "3.7"
+    services:
+    rems:
+        mem_reservation: 200m
+        mem_limit: 500m
+        ports:
+        - "127.0.0.1:3001:3000"
+        volumes:
+        - ./services/rems/simple-config.edn:/rems/config/config.edn
```

Comment:
I tried with these two additional volumes making up to these three with my directory structure:
```
        - ../rems/dev-config.edn:/rems/config/config.edn
        - ../rems/example-theme:/rems/config/example-theme
        - ../rems/test-data:/rems/test-data
```

The default config wants `ga4gh` keys and they live in `test-data` folder. Could perhaps do without them. Maybe including a complete config EDN file here makes sense? You have added the instructions later but I think it would be nice to see the end results.